### DISCUSSION TITLE
Session information not persisting

### DISCUSSION THREAD

**Discussion by mvyleteljr** [adonisjs/core#3406] [Help]
Hello!

I have been working at this bug for days and have made no progress. I have a 'nonce' value getting saved to session with session.put('nonce', nonce), and when i call session.get('nonce') on the next line, it outputs the expected behavior.

However, on subsequent requests, when calling session.get('nonce') I get that it is undefined. I have the configuration files setup to look like the demos with the cookie driver. 

Not sure how to make progress here, would love any pointers, please. Slightly time sensitive for investor meetings soon...

**Comment by thetutlage** [comment]
Can you create an example of this and share  it as a repo?

**Reply by mvyleteljr** [reply]
Sure will do that in a minute and drop the link here, in the meantime, I should note that sessionID info is persisting normally in cookie form and looks fine, but none of the other info is. Will be back shortly to share repo

**Reply by mvyleteljr** [reply]
https://github.com/mvyleteljr/adonis-share

**Reply by mvyleteljr** [reply]
to protect some api keys, i left out the .env file, but within it, SESSION_DRIVER = cookie to use the cookie driver

**Reply by mvyleteljr** [reply]
Also, the file where i session.put() is in the LoginController.ts file
